lib/cisco_node_utils/bfd_global.rb

```diff
@@ -33,9 +33,24 @@ def initialize(instantiate=true)
     # Reset everything back to default
     def destroy
       return unless Feature.bfd_enabled?
+      self.interval = default_interval if interval
       self.echo_interface = default_echo_interface
+      self.echo_rx_interval = default_echo_rx_interval if echo_rx_interval
+      self.ipv4_echo_rx_interval = default_ipv4_echo_rx_interval if
+        ipv4_echo_rx_interval
+      self.ipv6_echo_rx_interval = default_ipv6_echo_rx_interval if
+        ipv6_echo_rx_interval
       self.fabricpath_vlan = default_fabricpath_vlan if fabricpath_vlan
+      self.slow_timer = default_slow_timer
+      self.ipv4_slow_timer = default_ipv4_slow_timer if ipv4_slow_timer
+      self.ipv6_slow_timer = default_ipv6_slow_timer if ipv6_slow_timer
+      self.fabricpath_slow_timer = default_fabricpath_slow_timer if
+        fabricpath_slow_timer
       self.startup_timer = default_startup_timer if startup_timer
+      self.ipv4_interval = default_ipv4_interval if ipv4_interval
+      self.ipv6_interval = default_ipv6_interval if ipv6_interval
+      self.fabricpath_interval = default_fabricpath_interval if
+        fabricpath_interval
       set_args_keys_default
```

Comment:
There's a lot of code duplication here. I think it would be cleaner to do it this way:

```
    def destroy
        return unless Feature.bfd_enabled?
        [:interval,
         :echo_interface,
         :echo_rx_interval,
        ].each { |prop| self.send("#{prop}=", self.send("default_#{prop}")) }
    end
```